versioned_docs/version-3.3/advanced-solidus/promotions-system.mdx

```diff
@@ -342,3 +345,9 @@ the promotion has been created, add the _Payment_ rule and the _Half shipping_ a
 
 You can now go to the frontend and see how the shipment price is dropped by 50% if you select the
 configured payment method.
+
+## How-to guides
+
+- [How to use a custom promotion adjuster][how-to-use-a-custom-promotion-adjuster]
+
+[how-to-use-a-custom-promotion-adjuster]: /how-tos/how-to-use-a-custom-promotion-adjuster
```

Comment:
This made me think that our links from within versioned documentation pages will always point to the edge version of the reference. I guess we should always [use relative links](https://docusaurus.io/docs/markdown-features/links). cc @elia